trello_api_call.py

```diff
@@ -0,0 +1,39 @@
+import requests
```

Comment:
I think this file is unused now. In app.py, you are importing and using trello_items.py

So delete this file.